cdap-common/src/main/java/co/cask/cdap/common/metadata/AbstractMetadataClient.java

```diff
@@ -415,6 +415,16 @@ public void removeTag(EntityId id, String tag)
     removeTag(id.toMetadataEntity(), tag);
   }
 
+  /**
+   * @param metadataEntity the {@link MetadataEntity} for which to retrieve metadata tags across
```

Comment:
`the {@link MetadataEntity} for which to retrieve metadata tags across` -> `the {@link MetadataEntity} for which to retrieve the metadata properties`